integration/airflow/openlineage/airflow/extractors/extractors.py

```diff
@@ -27,26 +30,36 @@ class Extractors:
     """
     def __init__(self):
         # Do not expose extractors relying on external dependencies that are not installed
-        self.extractors = {
-            extractor.operator_class: extractor
-            for extractor
-            in _extractors
-            if getattr(extractor, 'operator_class', None) is not None
-        }
-
-        self.patchers = {
-            extractor.operator_class: extractor
-            for extractor
-            in _patchers
-            if getattr(extractor, 'operator_class', None) is not None
-        }
+        self.extractors = {}
+        self.patchers = {}
+
+        for extractor in _extractors:
+            for operator_class in extractor.get_operator_classnames():
+                self.extractors[operator_class] = extractor
+
+        for patcher in _patchers:
+            for operator_class in patcher.get_operator_classnames():
+                self.patchers[operator_class] = patcher
+
+        for key, value in os.environ.items():
```

Comment:
Mind if add a comment on the environment key pattern? For example: `OPENLINEAGE_EXTRACTOR_<extractor>`

Reply:
Added docs.

Reply:
Thanks!